python/src/synapse/ml/dl/DeepVisionModel.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from horovod.spark.lightning import TorchModel
+from pyspark.ml.param.shared import Param, Params
+from pyspark import keyword_only
+import torchvision.transforms as transforms
+from PIL import Image
+import torch
+from pyspark.sql.functions import udf, col
+from pyspark.sql.types import DoubleType
+import numpy as np
+
+
+class DeepVisionModel(TorchModel):
```

Comment:
Perhaps we should also extend SparkML Probablistic Classifier here and in the estimator

Reply:
Seems we can't inherit Probablistic Classification model here as it complains about `python cannot create consistent method ordering` , and this pobablistic Classifier also contains a complicated logic of different columns, so I'm going to extend the `_PredictorParams` directly and set the PredictionCol to the max value in model's outputCol.

Reply:
Ideally, we should persuade hovorod team to making HorovodModel extend PredictionModel.